tox.ini

```diff
@@ -27,12 +27,14 @@ commands=
     pytest -Wall --durations 25 {posargs}
 
 [testenv:flake8]
+basepython = python3
```

Comment:
Now we control the version of python by `.travis.yml`. I feel this setting is not needed.

Reply:
We control this for Travis but not for local runs. You'll see that I have two patches in this pull request: the first was visible for me because I'm using Fedora 28 which defaults to Python 3. I think tests and linters should run the same locally as they do in the CI, right?

Reply:
Ah, you're right. Go ahead.

BTW, flake8 is defined as `2.7` in `.travis.yml`. Should this be also changed?
https://github.com/sphinx-doc/sphinx/blob/f14d60dcd2772b7e973cc7dee7a6e625b44926c6/.travis.yml#L33-L34

Reply:
Yes, I will change this shortly.